draft-ietf-acme-scoped-dns-challenges.mkd

```diff
@@ -66,15 +66,11 @@ Furthermore, since each domain only has a single authorization label, it creates
 
 In multi-region deployments, where separate availability zones serve the same content, and dependencies across them are avoided, operators need a way to obtain a separate certificate per zone, for the same domain name. Similarly, in cases of zero-downtime migration, two different setups of the infrastructure may coexist for a long period of time, and both need access to valid certificates.
 
-This document specifies two new challenge types. `dns-02` and `dns-account-01`, which aim to address these deficiencies.
+This document specifies the `dns-account-01` validation method, which aim to address these deficiencies.
 
 This work follows all recommendations set forth in "Domain Control Validation using DNS" [I-D.draft-ietf-dnsop-domain-verification-techniques].
 
-This RFC does not intend to deprecate the `dns-01` challenge specified in {{!RFC8555}}. Since these new challenges do not modify any pre-existing challenges, the ability to complete the `dns-02` or `dns-account-01` challenge requires ACME server operators to deploy new changes to their codebase. This makes adopting and using these challenges an opt-in process.
-
-## DNS-02
-
-The `dns-02` challenge adds onto `dns-01` by introducing a scoping mechanism to the domain authorization label. This allows for the client to specify if the intended domain validation is for a specific host, a wildcard domain, or a domain and all of its subdomains.
+This RFC does not intend to deprecate the `dns-01` challenge specified in {{!RFC8555}}. Since these new challenges do not modify any pre-existing challenges, the ability to complete the `dns-account-01` challenge requires ACME server operators to deploy new changes to their codebase. This makes adopting and using these challenges an opt-in process.
```

Comment:
```suggestion
This RFC does not intend to deprecate the `dns-01` challenge specified in {{!RFC8555}}. The `dns-account--01` challenge requires ACME server operators to deploy new changes to their codebase and is an opt-in process.
```

Reply:
Plural "challenges" was used a few times in these sentences. I also find it a bit awkward so I tried to simplify the text without losing meaning. I think there is still room for improvement on it. Feel free to further edit.